Editorial review: Document ViewTransition.waitUntil() (#43962)
mdn/content: files/en-us/web/api/view_transition_api/using/index.md

   At this point, the view transition is about to run, and the {{domxref("ViewTransition.ready")}} promise fulfills, allowing you to respond by running a custom JavaScript animation instead of the default, for example.

5. The old page snapshots animate "out", while the new view snapshots animate "in". By default, the old view snapshots animate from {{cssxref("opacity")}} 1 to 0, and the new view snapshots animate from `opacity` 0 to 1, which creates a cross-fade.

> 6. When the transition animations have reached their end states, the snapshots are destroyed, and the {{domxref("ViewTransition.finished")}} promise fulfills, allowing you to respond. You can delay a view transition from reaching its finished state until a specified {{domxref("Promise")}} is resolved using the {{domxref("ViewTransition.waitUntil()")}} method, which has utility in advanced use cases.

Reviewer: I'd be tempted to omit the "advanced use cases".

```suggestion
6. When the transition animations have reached their end states, the snapshots are destroyed, and the {{domxref("ViewTransition.finished")}} promise fulfills, allowing you to respond. If needed, you can delay a view transition from reaching its finished state until a specified {{domxref("Promise")}} is resolved using the {{domxref("ViewTransition.waitUntil()")}} method.
```

> 6. When the transition animations have reached their end states, the snapshots are destroyed and the {{domxref("ViewTransition.finished")}} promise fulfills, allowing you to respond. You can delay a view transition from getting to its finished state until a specified {{domxref("Promise")}} is resolved, using the {{domxref("ViewTransition.waitUntil()")}} method, which has some utility in advanced use cases.

Reviewer 2: ```suggestion
6. When the transition animations have reached their end states, the snapshots are destroyed, and the {{domxref("ViewTransition.finished")}} promise fulfills, allowing you to respond. You can delay a view transition from reaching its finished state until a specified {{domxref("Promise")}} is resolved using the {{domxref("ViewTransition.waitUntil()")}} method, which has utility in advanced use cases.
```
